core/src/View/Kokkos_BasicView.hpp

```diff
@@ -491,46 +483,24 @@ class BasicView {
              (sizeof...(OtherIndexTypes) == rank()))
   KOKKOS_FUNCTION constexpr reference operator()(
       OtherIndexTypes... indices) const {
+    KOKKOS_IMPL_BASICVIEW_OPERATOR_VERIFY(indices...);
     return m_acc.access(m_ptr,
                         m_map(static_cast<index_type>(std::move(indices))...));
   }
-
-  template <class OtherIndexType>
-    requires(
-        std::is_convertible_v<const OtherIndexType &, index_type> &&
-        std::is_nothrow_constructible_v<index_type, const OtherIndexType &>)
-  KOKKOS_FUNCTION constexpr reference operator()(
-      const Array<OtherIndexType, rank()> &indices) const {
-    return m_acc.access(m_ptr,
-                        [&]<size_t... Idxs>(std::index_sequence<Idxs...>) {
-                          return m_map(indices[Idxs]...);
-                        }(std::make_index_sequence<rank()>()));
-  }
-
-  template <class OtherIndexType>
-    requires(
-        std::is_convertible_v<const OtherIndexType &, index_type> &&
-        std::is_nothrow_constructible_v<index_type, const OtherIndexType &>)
-  KOKKOS_FUNCTION constexpr reference operator()(
-      std::span<OtherIndexType, rank()> indices) const {
-    return m_acc.access(m_ptr,
-                        [&]<size_t... Idxs>(std::index_sequence<Idxs...>) {
-                          return m_map(indices[Idxs]...);
-                        }(std::make_index_sequence<rank()>()));
-  }
```

Comment:
Why did you remove these functions?

Reply:
They aren't needed as part of the view interface (they are part of the mdspan interface). Also I think they caused CUDA 11.x problems...

Reply:
Fine